src/main/java/uk/gov/justice/digital/hmpps/whereabouts/model/VideoLinkBooking.kt

```diff
@@ -0,0 +1,31 @@
+package uk.gov.justice.digital.hmpps.whereabouts.model
+
+import javax.persistence.CascadeType.PERSIST
+import javax.persistence.CascadeType.REMOVE
+import javax.persistence.Entity
+import javax.persistence.GeneratedValue
+import javax.persistence.GenerationType
+import javax.persistence.Id
+import javax.persistence.JoinColumn
+import javax.persistence.ManyToOne
+import javax.persistence.Table
+
+@Entity
+@Table(name = "VIDEO_LINK_BOOKING")
+data class VideoLinkBooking(
+  @Id
+  @GeneratedValue(strategy = GenerationType.IDENTITY)
+  val id: Long? = null,
```

Comment:
Shouldn't be nullable.

Reply:
This has to be nullable because a transient VLB has no ID. That's a Hibernate requirement (I think)